src/libraries/Native/Unix/System.Native/pal_signal.c

```diff
@@ -106,21 +162,7 @@ static void* SignalHandlerLoop(void* arg)
             }
         }
 
-        if (signalCode == SIGQUIT || signalCode == SIGINT)
-        {
-            // We're now handling SIGQUIT and SIGINT. Invoke the callback, if we have one.
-            CtrlCallback callback = g_ctrlCallback;
-            CtrlCode ctrlCode = signalCode == SIGQUIT ? Break : Interrupt;
-            if (callback != NULL)
-            {
-                callback(ctrlCode);
-            }
-            else
-            {
-                SystemNative_RestoreAndHandleCtrl(ctrlCode);
-            }
-        }
-        else if (signalCode == SIGCHLD)
+        if (signalCode == SIGCHLD)
```

Comment:
note: we don't allow to cancel the `SIGCHLD` processing that is used by the `Process` class.